src/librustc_codegen_ssa/back/link.rs

```diff
@@ -158,6 +159,33 @@ pub fn get_linker(sess: &Session, linker: &Path, flavor: LinkerFlavor) -> (PathB
         }
     };
 
+    let t = &sess.target.target;
```

Comment:
Could you include a comment above this block indicating why it exists, what it's loading, etc?